.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,7 @@ Pre-checks:
 * [ ] Changelog entry added
 * [ ] Tests have been added (if applicable)
 * [ ] Branch prefixed with `feature/`, `hotfix/` or `spike/`
+* [ ] Docs ticket created
```

Comment:
```suggestion
* [ ] Docs ticket created (if applicable)
```
This isn't going to be required for every PR. Also should this be in `Review` instead? Because it's not something you really want done before the PR is logged.